src/nvidia_rag/utils/summary_status_handler.py

```
@@ -0,0 +1,267 @@
+            self._redis_available = True
+            logger.info(
+                f"Connected to Redis at {self._redis_host}:{self._redis_port} "
+                f"(pool: {REDIS_MAX_CONNECTIONS})"
+            )
+        except (ConnectionError, RedisError, OSError) as e:
+            self._redis_available = False
+            self._redis_pool = None
+            self._redis_client = None
+            logger.warning(
+                f"Redis unavailable at {self._redis_host}:{self._redis_port} - "
+                f"summary status tracking disabled: {e}"
```

Ruff (0.14.5)

Logging statement uses f-string

(G004)